specification/communication/data-plane/PhoneNumbers/preview/2022-06-01-preview/phonenumbers.json

```diff
@@ -0,0 +1,917 @@
+        "id",
+        "operationType",
+        "status"
+      ],
+      "type": "object",
+      "properties": {
+        "status": {
+          "description": "Status of operation.",
+          "enum": [
+            "notStarted",
+            "running",
+            "succeeded",
+            "failed"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberOperationStatus",
+            "modelAsString": true
+          }
+        },
+        "resourceLocation": {
+          "description": "URL for retrieving the result of the operation, if any.",
+          "type": "string"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "description": "The date that the operation was created.",
+          "type": "string"
+        },
+        "error": {
+          "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationError"
+        },
+        "id": {
+          "description": "Id of operation.",
+          "type": "string"
+        },
+        "operationType": {
+          "description": "The type of operation, e.g. Search",
+          "enum": [
+            "purchase",
+            "releasePhoneNumber",
+            "search",
+            "updatePhoneNumberCapabilities"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberOperationType",
+            "modelAsString": true
+          }
+        },
+        "lastActionDateTime": {
+          "format": "date-time",
+          "description": "The most recent date that the operation was changed.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "PhoneNumberCapabilitiesRequest": {
+      "description": "Capabilities of a phone number.",
+      "type": "object",
+      "properties": {
+        "calling": {
+          "description": "Capability value for calling.",
+          "enum": [
+            "none",
+            "inbound",
+            "outbound",
+            "inbound+outbound"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberCapabilityType",
+            "modelAsString": true
+          }
+        },
+        "sms": {
+          "description": "Capability value for SMS.",
+          "enum": [
+            "none",
+            "inbound",
+            "outbound",
+            "inbound+outbound"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberCapabilityType",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "PurchasedPhoneNumber": {
+      "description": "Represents a purchased phone number.",
+      "required": [
+        "assignmentType",
+        "capabilities",
+        "cost",
+        "countryCode",
+        "id",
+        "phoneNumber",
+        "phoneNumberType",
+        "purchaseDate",
+        "partnerName",
+        "partnerId",
+        "isOperatorConnect"
```

Comment:
Can this be a more descriptive name of the parameter?

Reply:
@MushMal would 'IsOperatorConnectNumber' be more descriptive? If not, can you elaborate more please?

Reply:
> 

I like the idea of making this boolean an enum - easier to extend. Make sure you are using extensible enums https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#use-extensible-enums

The name itself can be anything like you have - phoneNumberType or phoneNumberSource..?

Reply:
I see you've made the changes already. Looks good

Reply:
Hi @JeffreyRichter and @MushMal can this be merged? Thanks!

Reply:
Apologies for the delay. Let me figure this out ASAP as I don't seem to have the merge rights either

Reply:
Hi @MushMal any eta on when this can be merged please? Thanks!